src/compass/defaults/s1_cslc_radar.yaml

```
@@ -68,6 +68,11 @@ runconfig:
            resample:
                lines_per_block: 1000
                flatten: False
+           correction_luts:
+               # LUT spacing in x/slant range direction in meters
+               x_spacing: 500
+               # LUT spacing in y/azimuth direction in seconds
+               y_spacing: 50
```

Is the LUT in radargrid? Then I would suggest to put more reasonable values here. Maybe the same as `s1_cslc_geo.yaml`
```suggestion
           correction_luts:
               # LUT spacing in x/slant range direction in meters
               x_spacing: 120
               # LUT spacing in y/azimuth direction in seconds
               y_spacing: 0.028
```